src/core/lib/transport/metadata.h

```diff
@@ -85,12 +85,12 @@ typedef enum {
   GRPC_MDELEM_STORAGE_EXTERNAL = 0,
   /* memory pointed to by grpc_mdelem::payload is interned by the metadata
      system */
-  GRPC_MDELEM_STORAGE_INTERNED = GRPC_MDELEM_STORAGE_INTERNED_BIT,
+  GRPC_MDELEM_STORAGE_STATIC = GRPC_MDELEM_STORAGE_INTERNED_BIT,
```

Comment:
This was intentional - it's for slightly better code generation for the switch() cases. No tests fail since the semantics of the enum have not changed; all we want is for GRPC_MDELEM_STORAGE_STATIC and GRPC_MDELEM_STORAGE_INTERNED to be different values which nonetheless have GRPC_MDELEM_STORAGE_INTERNED_BIT set; and flipping the order does not change any of the expectations.

In other words: there is nothing to test here since we didn't change anything in the high level semantics; only we improved slightly the resulting code generation.

Reply:
Please change the comments above them to reflect the update

Reply:
Please change the comment above GRPC_MDELEM_STORAGE_STATIC and GRPC_MDELEM_STORAGE_INTERNED to reflect the update.

Also, consider leaving a comment about the code generation optimization so that others know what to look for when modifying this code in the future.